### ISSUE TITLE
smarter restart

### ISSUE THREAD

**Issue by Almenon** [Almenon/AREPL-vscode#236]

**Comment by Almenon** [comment]
One problem with this is that I like to be able to simply add a space to trigger a new run of arepl (for code that changes between runs for whatever reason). There is a hotkey to trigger a new run (control-shift-;) but I don't want to have to remember a seldom used hotkey. I suppose implementing this issue would be okay if there was a run button somewhere. 

Unfortunately not sure where to put the run button and a run button means there would also need to be a "running" indicator and a stop button and I need to think about how it would work in inline mode too ..... ugg

**Comment by goyalyashpal** [comment]
> There is a hotkey to trigger a new run (control-shift-;) but I don't want to have to remember a seldom used hotkey.

yeah, restarting arepl is a pain especially because of the need to require an edit on ctrl-shift-a
